shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -597,26 +597,36 @@ protected void setWindowInfoListenerDisplayFeatures(WindowLayoutInfo layoutInfo)
   // android may decide to place the software navigation bars on the side. When the nav
   // bar is hidden, the reported insets should be removed to prevent extra useless space
   // on the sides.
-  private enum ZeroSides {
+  @VisibleForTesting
+  public enum ZeroSides {
     NONE,
     LEFT,
     RIGHT,
     BOTH
   }
 
-  private ZeroSides calculateShouldZeroSides() {
+  /**
+   * This method can be run on APIs 30 and above but its intended use is for 30 and below.
+   *
+   * @return some ZeroSides enum
+   */
+  @SuppressWarnings("deprecated")
```

Comment:
fyi I think this needs to be
```suggestion
  @SuppressWarnings("deprecation")
```

Reply:
at least I assume that's why the lint check is failing

Reply:
done. Changed baseline.xml file to suppress linter